locale/ar/get-involved/contribute.md

```diff
@@ -13,12 +13,12 @@ layout: contribute.hbs
 
 ## الإبلاغ عن مشكلة
 
-إذا وجدت أي شيء تعتقد انه مشكلة في الـ Node.js، لا تتردد في في فتح مشكلة على المشروع في الـ GitHub. عند طرح للمشكلة، احرص على ان تعبر عنها بفحوصات قابلة للاعادة، كما يجب على تلك الفحوصات ألا تستعين بأي مكتبات خارجية. بشكل أوضح، يجب على تلك الفحوصات ان تكون قابلة للتنفيذ اعتماداً على الـ Node.js لا أكثر.
+إذا وجدت أي شيء تعتقد انه مشكلة في الـ Node.js، لا تتردد في في فتح مشكلة على المشروع في الـ GitHub. عند طرح مشكلتك، احرص على ان تعبر عنها بفحوصات قابلة للاعادة، كما يجب على تلك الفحوصات ألا تستعين بأي مكتبات خارجية. بشكل أوضح، يجب على تلك الفحوصات ان تكون قابلة للتنفيذ اعتماداً على الـ Node.js لا أكثر.
 
 عند التبليغ عن مشكل، نحتاج اكبر قدر ممكن من المعلومات حول بيئة التشغيل  التي تستعملها. لا ندري أي معلومة لها صلة وثيقة بالمشكلة عند محاولة حصر المشكلة، قم بتضمين المعلومات التالية على الأقل:
 
 * إصدار الـ Node.js
-* منصة التشغيل التي تعمل عليها (macOS, SunOS, Linux, Windows)
+* منصة التشغيل التي تعمل عليها (macOS, SmartOS, Linux, Windows)
```

Comment:
Why change SunOS to SmartOs ?

Reply:
https://nodejs.org/en/get-involved/contribute/ 

![image](https://user-images.githubusercontent.com/43971542/63639441-6516dc80-c693-11e9-9989-8327e3a31b03.png)

Reply:
Sorry, I had not seen the original (english version).